bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/discovery/SomfyTahomaItemDiscoveryService.java

```diff
@@ -366,8 +386,17 @@ private boolean hasCommmand(SomfyTahomaDevice device, String command) {
         return device.getDefinition().getCommands().stream().anyMatch(cmd -> command.equals(cmd.getCommandName()));
     }
 
-    private void deviceDiscovered(SomfyTahomaDevice device, ThingTypeUID thingTypeUID) {
-        deviceDiscovered(device.getLabel(), device.getDeviceURL(), device.getOid(), thingTypeUID,
+    private void deviceDiscovered(SomfyTahomaDevice device, ThingTypeUID thingTypeUID, @Nullable String place) {
+        deviceDiscovered(null, device, thingTypeUID, place);
+    }
+
+    private void deviceDiscovered(@Nullable String otherLabel, SomfyTahomaDevice device, ThingTypeUID thingTypeUID,
+            @Nullable String place) {
```

Comment:
Why is here the method ```deviceDiscovered``` with first parameter ```@Nullable String otherLabel```, I see calling only with the ```null``` value. In this case merge this method with ```private void deviceDiscovered(SomfyTahomaDevice device, ThingTypeUID thingTypeUID, @Nullable String place)``` and have one method instead.

Reply:
I was expecting this remark ;)
I used it in coming code (next PR).
But I could introduce it only in my next PR.

Reply:
Fixed. Method removed.